src/ed_object_models/spawn_sdf_in_gazebo.py

```diff
@@ -45,9 +39,9 @@ def from_yaml(yaml_file, package_name='fast_simulator_data'):
     for item in items:
         # Define object pose
         object_pose = Pose()
-        object_pose.position.x = item['x']
-        object_pose.position.y = item['y']
-        object_pose.position.z = item['z']
+        object_pose.position = Point(item['x'], item['y'], item['z'])
+        if ('roll' in item) & ('pitch' in item) & ('yaw' in item):    # Check if euler angle fields exist
```

Comment:
use `AND`.

Would be nice if it also works if some fields are missing. Take zero as default

Reply:
I have added this.